analyses/pypi/pypi.go

```diff
@@ -0,0 +1,64 @@
+package pypi
+
+// This File contains Utility functions of Pypi Ecosystem
+
+import (
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+
+	"github.com/rs/zerolog/log"
+)
+
+// checkName checks for valid file name.
+func checkName(name string) bool {
+	match1, _ := regexp.MatchString("requirements?", name)
+	match2, _ := regexp.MatchString("constraints?", name)
+	if match1 || match2 {
+		return true
+	}
+	return false
+}
+
+// checkExt checks for valid file extension.
+func checkExt(ext string) bool {
+	switch ext {
+	case
+		".in",
+		".txt":
+		return true
+	}
+	return false
+}
+
+// getPylistGenerator generates `generate_pylist.py from `generatepylist.go`
+func (m *Matcher) getPylistGenerator() error {
+	log.Debug().Msgf("Executing: getPylistGenerator")
+	// Generating generate_pylist.py
+	err := ioutil.WriteFile("generate_pylist.py", codeForPylist, 0644)
```

Comment:
So this writes into current working dir? This should be avoided and written into tmp dir.

Reply:
Ack. Shifted base to /tmp